scripts/cnode-helper-scripts/mithril-signer.sh

```diff
@@ -225,9 +224,18 @@ else
     # Run Mithril Signer Server
     echo "Starting Mithril Signer Server.."
     trap 'user_interrupt_received' INT
-    if ! "${MITHRILBIN}" -vv | tee -a "${LOG_DIR}/$(basename "${0::-3}")".log 2>&1 ; then
-      echo "Failed to start Mithril Signer Server" | tee -a "${LOG_DIR}/$(basename "${0::-3}")".log 2>&1
-      exit 1
+
+    if grep -q "ENABLE_METRICS_SERVER=true" ${CNODE_HOME}/mithril/mithril.env; then
+      METRICS_SERVER_PARAMS="--enable-metrics-server --metrics-server-ip ${METRICS_SERVER_IP} --metrics-server-port ${METRICS_SERVER_PORT}"
+      if ! "${MITHRILBIN}" ${METRICS_SERVER_PARAMS} -vv | tee -a "${LOG_DIR}/$(basename "${0::-3}")".log 2>&1 ; then
```

Comment:
Enables gLiveView automatically. We could also inquire with an additional question if they want it enabled. I think when reconfiguring the environment file automatically haveing gLiveView show the signer might be useful to show if a misconfiguration exists and the signer does not start, the SPO can always disable it again in the env file.

```suggestion
    if grep -q "ENABLE_METRICS_SERVER=true" ${CNODE_HOME}/mithril/mithril.env; then
      METRICS_SERVER_PARAMS="--enable-metrics-server --metrics-server-ip ${METRICS_SERVER_IP} --metrics-server-port ${METRICS_SERVER_PORT}"
      # If ENABLE_METRICS_SERVER is true, then an environment update will enable gLiveView automatically.
      sudo sed -i "s/#MITHRIL_SIGNER_ENABLED=[YN]/MITHRIL_SIGNER_ENABLED=Y/" ${CNODE_HOME}/scripts/env
      if ! "${MITHRILBIN}" ${METRICS_SERVER_PARAMS} -vv | tee -a "${LOG_DIR}/$(basename "${0::-3}")".log 2>&1 ; then
```